docs/user-manual/graphics/shaders/wgsl-specifics.md

```diff
@@ -3,4 +3,273 @@ title: WGSL Specifics
 sidebar_position: 2
 ---
 
-Support for WGSL in the PlayCanvas engine is under active development. Detailed information on WGSL shader requirements and integration will be provided once the implementation is finalized.
+WGSL shaders used by the PlayCanvas engine must satisfy certain requirements. These requirements allow the engine to correctly integrate shaders, ensuring they receive the necessary resources such as attributes, uniforms, and varyings.
+
+The following sections outline key aspects of writing WGSL shaders for PlayCanvas.
+
+### Simplified Shader Interface Syntax
+
+In standard WGSL (WebGPU Shading Language), declaring uniforms, attributes, and varyings requires explicitly specifying a @group and @binding index for each resource. This can be verbose and error-prone, especially for common patterns.
```

Comment:
```suggestion
In standard WGSL (WebGPU Shading Language), declaring uniforms, attributes, and varyings requires explicitly specifying a `@group` and `@binding` index for each resource. This can be verbose and error-prone, especially for common patterns.
```